units/systemd-journald-varlink@.socket

```diff
@@ -8,7 +8,7 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=Journal Varlink Socket for Namespace %i
```

Comment:
Hmm, I think we should keep this one because `systemd-journald@.socket` exists.

Reply:
Indeed, updated.